torch/jit/_tuples.py

```diff
@@ -0,0 +1,2 @@
+# This is a dummy module where we put all of the known NamedTuple types so that they can be
```

Comment:
discussion offline: We need to have namedtuple return the Python equivalent tuple, if it exists, so that pickle will work. This mechanism won't work if the loading process is not the same as the saving process.